Traceback (most recent call last):
  File "/home/ale/TT_RLProject_2024/Task_6/UDR_Training.py", line 74, in <module>
    main()
  File "/home/ale/TT_RLProject_2024/Task_6/UDR_Training.py", line 44, in main
    env = DummyVecEnv([make_env(config['env_name'])])
  File "/home/ale/miniconda3/envs/mldl/lib/python3.7/site-packages/stable_baselines3/common/vec_env/dummy_vec_env.py", line 26, in __init__
    self.envs = [fn() for fn in env_fns]
  File "/home/ale/miniconda3/envs/mldl/lib/python3.7/site-packages/stable_baselines3/common/vec_env/dummy_vec_env.py", line 26, in <listcomp>
    self.envs = [fn() for fn in env_fns]
TypeError: 'Monitor' object is not callable